sql/src/test/java/io/crate/protocols/postgres/types/PGArrayTest.java

```diff
@@ -66,57 +66,20 @@ public void testArrayWithNullValues() {
         assertThat(o, is(array));
     }
 
-    @SuppressWarnings("unchecked")
-    @Test
-    public void testJsonArrayEncodeDecode() {
-        // 1-dimension array
-        // Decode
-        String s = "{\"{\"names\":[\"Arthur\",\"Trillian\"]}\",\"{\"names\":[\"Ford\",\"Slarti\"]}\"}";
-        List<Object> values = (List<Object>) PGArray.JSON_ARRAY.decodeUTF8Text(s.getBytes(StandardCharsets.UTF_8));
-
-        List<String> names = (List<String>) ((Map) values.get(0)).get("names");
-        assertThat(names, Matchers.contains("Arthur", "Trillian"));
-
-        names = (List<String>) ((Map) values.get(1)).get("names");
-        assertThat(names, Matchers.contains("Ford", "Slarti"));
-
-        // Encode
-        byte[] bytes = PGArray.JSON_ARRAY.encodeAsUTF8Text(values);
-        s = new String(bytes, StandardCharsets.UTF_8);
-        assertThat(s, is(s));
```

Comment:
this and the following assertions will always pass and was never asserted correctly. Also, the string `s` contains the format that could not be produced with `encodeAsUTF8Text`. I replaced this test cases with a single encode/decode round-trip case.